server/src/index.js

```diff
@@ -32,6 +35,7 @@ server.use("/api/testPush", TestPushRouter);
 server.use("/api/populate", PopulateRouter);
 server.use("/api/cacheFavorites", CacheFavoritesRouter);
 server.use("/api/preferences", PreferencesRouter);
+server.use("/api/cacheHours", CacheHoursRouter);
```

Comment:
Do we really need another route for this? I'm thinking we're starting to suffer a bit from route bloat and that we should put it all (or at least the daily ones) under the same route. At the same time, we want to be sure that any one of them throwing doesn't prevent execution of the others, so we need to be careful about how we code it.